tests/legacy-cli/e2e/initialize/500-create-project.ts

```diff
@@ -22,6 +22,10 @@ export default async function () {
     // Ensure local test registry is used when outside a project
     await setNPMConfigRegistry(true);
 
+    // Install puppeteer in the parent directory of the test projects for use
+    // by the CLI within any test project
+    await installPackage('puppeteer@14.1.1');
```

Comment:
NIT: would be great if we can use the same version of `package.json`. This would keep it in sync as otherwise I feel that we'll forget to update this.

Reply:
SGTM.